src/types/onyx/ReimbursementAccount.ts

```diff
@@ -8,7 +8,7 @@ type BankAccountSubStep = ValueOf<typeof CONST.BANK_ACCOUNT.SUBSTEP>;
 
 type ACHData = {
     /** Step of the setup flow that we are on. Determines which view is presented. */
-    currentStep: BankAccountStep;
+    currentStep?: BankAccountStep;
```

Comment:
Why this was changed?

Reply:
`reimbursementAccountDefaultProps` do not have this field inside so I dont think it should be required